startup_scripts/210_virtualization_interfaces.py

```diff
@@ -0,0 +1,47 @@
+from dcim.models import Interface
+from virtualization.models import VirtualMachine
+from extras.models import CustomField, CustomFieldValue
+from ruamel.yaml import YAML
+
+from pathlib import Path
+import sys
+
+file = Path('/opt/netbox/initializers/virtualization_interfaces.yml')
+if not file.is_file():
+  sys.exit()
+
+with file.open('r') as stream:
+  yaml = YAML(typ='safe')
+  interfaces = yaml.load(stream)
+
+  optional_assocs = {
```

Comment:
Is this really optional? An interface without a VM does not make sense to me, or am I missing something?